src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -212,6 +212,14 @@ class CommandRequestImpl : public CommandImpl,
   bool HasDisallowedParams() const;
 
   /**
+   * @brief Checks if file name contains invalid symbols e.g. '/'
+   * @param file_name file name to check
+   * @return true if file name does not contain any invalid symbol otherwise
+   * returns false
+   */
+  bool IsFileNameValid(const std::string& file_name) const;
```

Comment:
@LuxoftAKutsan yes, it is. Was moved to `file_system` utils functions. See https://github.com/smartdevicelink/sdl_core/pull/1613/commits/f050b8f14ed2f3e048d7a4a0b2f1c424255649ac